### DISCUSSION TITLE
[A-L] (2024/25) Foglio 2 - Esercizio 2

### DISCUSSION THREAD

**Discussion by Rurik-D** [sapienzastudentsnetwork/algebra#481] [Esercizi - Viaggi/Cherubini]
![image](https://github.com/user-attachments/assets/41eb8864-9418-453d-ab6d-22c1f26fb735)

**Comment by matypist** [comment]
Soluzione ad opera della [tutor](https://github.com/sapienzastudentsnetwork/algebra/blob/main/scandiuzzi/README.md#aa-202425-scandiuzzi-tutor) Elisa **Scandiuzzi**, tutoraggio del **9 ottobre** 2024 ([appunti_24_10_09.pdf](https://drive.google.com/file/d/1tqiXXi2WFl-N2J1HJcQ47xS_EOnubLaC/view?usp=drive_link)):

![image](https://github.com/user-attachments/assets/633f7eb8-39f8-4a78-9abd-5e1a5a3c799c)
![image](https://github.com/user-attachments/assets/f1f57953-b865-46b9-93db-d22f4fc7ea8d)
![image](https://github.com/user-attachments/assets/800a2186-9e28-4372-98e1-b28706de833d)
